.vscode/cspell.json

```diff
@@ -21,13 +21,52 @@
     "*.sha512",
     "*.ico",
     "*.pri",
+    "*.trx",
+    "*.binlog",
+    "*.log",
+    "*.sln",
+    "*.user",
     ".vscode/cspell.json"
   ],
   // cspell is not case sensitive
   // Sort words alphabetically to make this list easier to use
-  "words": ["azsdk", "pwsh"],
+  "words": [
+    "apos",
+    "azsdk",
+    "blazor",
+    "contoso",
+    "deserializes",
+    "diagnoser",
+    "epsg",
+    "expando",
+    "finalizer",
+    "finalizers",
+    "ints",
+    "lucene",
+    "mgmt",
+    "nunit",
+    "odata",
+    "otel",
+    "overridden",
+    "parallelization",
+    "pkcs",
+    "pwsh",
+    "referer",
+    "renormalize",
+    "retriable",
+    "struct",
```

Comment:
`struct` is a C# keyword and, using the configuration currently in the `master` branch the word `struct` raises a spelling error in `.md` files but NOT in `.cs` files (that is, `languageId` applies the `csharp` dictionary to files it thinks are C# and doesn't apply that dictionary to files that are not C#). 

To avoid bloating `words`, I recommend adding `csharp` to the `dictionaries` list (line 5) to include C# specific terms. 

Looks like we can avoid a lot of false positives by adding this to the other repos' configurations as well. I'll open up work items for those.